files/plugins/cron_cis_audit.py

```diff
@@ -39,45 +40,53 @@ def _get_major_version():
 else:
     AUDIT_FOLDER = "/var/lib/usg/"
     AUDIT_RESULT_GLOB = AUDIT_FOLDER + "/usg-results-*.*.xml"
-    PROFILES = ["cis_" + p for p in PROFILES]
     AUDIT_BIN = ["/usr/sbin/usg", "audit"]
 
 
 CLOUD_INIT_LOG = "/var/log/cloud-init-output.log"
-DEFAULT_PROFILE = PROFILES[0]
 
 MAX_SLEEP = 600
 PID_FILENAME = "/tmp/cron_cis_audit.pid"
 TAILORING_CIS_FILE = Path("/etc/usg/default-tailoring.xml")
 
 
 def _get_cis_hardening_profile(profile):
-    """Try to read the cis profile from cloud init log or default to level1_server.
+    """Try to read the cis profile from cloud init log or defaults to the first option.
 
     If the "default" tailoring file exists in /etc/usg/default-tailoring.xml,
     no profile is passed.
     """
     if TAILORING_CIS_FILE.exists():
         return None
 
-    if profile in PROFILES:
+    profiles = _get_profile_by_ubuntu_series()
+    default_profile = profiles[0]
+
+    if profile in profiles:
         return profile
 
     if not os.path.exists(CLOUD_INIT_LOG) or not os.access(CLOUD_INIT_LOG, os.R_OK):
         print(
             "{} not existing/accessible, default to profile '{}'".format(
-                CLOUD_INIT_LOG, DEFAULT_PROFILE
+                CLOUD_INIT_LOG, default_profile
             )
         )
-        return DEFAULT_PROFILE
+        return default_profile
 
     pattern = re.compile(r"Applying Level-(1|2) scored (server|workstation)")
     for _, line in enumerate(open(CLOUD_INIT_LOG)):
         for match in re.finditer(pattern, line):
             level, machine_type = match.groups()
             return "level{}_{}".format(level, machine_type)
 
-    return DEFAULT_PROFILE
+    return default_profile
+
+
+def _get_profile_by_ubuntu_series():
```

Comment:
Maybe something like this, to be clearer that it gets a list of profiles, and it's dependent on the current running system?

```suggestion
def _valid_profiles_for_platform():
```